authentication/passwordless/passwordless.go

```diff
@@ -0,0 +1,57 @@
+package passwordless
+
+import "github.com/auth0/go-auth0/authentication/oauth"
+
+// SendEmailRequest defines the request body for starting a passwordless flow via email.
+type SendEmailRequest struct {
+	oauth.ClientAuthentication
+	Connection string                 `json:"connection,omitempty"`
+	Email      string                 `json:"email,omitempty"`
+	Send       string                 `json:"send,omitempty"`
+	AuthParams map[string]interface{} `json:"authParams,omitempty"`
+}
+
+// SendEmailResponse defines the response from the `SendEmail` request.
+type SendEmailResponse struct {
+	ID            string `json:"_id,omitempty"`
+	Email         string `json:"email,omitempty"`
+	EmailVerified bool   `json:"email_verified,omitempty"`
+}
+
+// LoginWithEmailRequest defines the request body for exchanging a code requested by `SendEmail` for a token.
+type LoginWithEmailRequest struct {
+	oauth.ClientAuthentication
+	Audience  string `json:"audience,omitempty"`
+	Code      string `json:"otp,omitempty"`
+	Email     string `json:"username,omitempty"`
+	GrantType string `json:"grant_type,omitempty"`
+	Realm     string `json:"realm,omitempty"`
+	Scope     string `json:"scope,omitempty"`
+}
+
+// SendSMSRequest defines the request body for starting a passwordless flow via email.
+type SendSMSRequest struct {
+	oauth.ClientAuthentication
+	Connection  string                 `json:"connection,omitempty"`
+	PhoneNumber string                 `json:"phone_number,omitempty"`
+	AuthParams  map[string]interface{} `json:"authParams,omitempty"`
+}
+
+// SendSMSResponse defines the response from the `SendSMS` request.
+type SendSMSResponse struct {
+	ID              string `json:"_id,omitempty"`
+	PhoneNumber     string `json:"phone_number,omitempty"`
+	PhoneVerified   bool   `json:"phone_verified,omitempty"`
+	RequestLanguage string `json:"request_language,omitempty"`
```

Comment:
Is this a field present in the response for "send SMS"?

Reply:
(just checking, because it's not present in the response for "send email").

Reply:
I included this on auth0-java and auth0.net including it, but given that I haven't been able to verify it via testing or reading the code I think I'll drop it and potentially add back in future if it does exist